### PR TITLE
Add support for kafka as log backend for verification

### PR DESCRIPTION
**Pull request by smalleni**
Signed-off-by: Sai Sindhur Malleni <smalleni@redhat.com>

### CODE REVIEW

File: roles/log_generator/templates/log_generator.yml [hunk 1]
```diff
@@ -106,10 +106,14 @@ spec:
             --aws-region "{{ workload_args.aws_region }}"
             --aws-access-key "{{ workload_args.aws_access_key }}"
             --aws-secret-key "{{ workload_args.aws_secret_key }}"
+{% endif %}
+{% if workload_args.kafka_bootstrap_server is defined and workload_args.kafka_topic is defined %}
+            --kafka-bootstrap-server "{{ workload_args.kafka_bootstrap_server }}"
+            --kafka-topic "{{  workload_args.kafka_topic }}"
```
[the remark this answers is not in the document]
Comment (dry923) [reply roles/log_generator/templates/log_generator.yml:112]: :muscle:

**Comment by smalleni** [comment]
@dry923 Good to review now

**Review by jtaleric — APPROVED** [review]
LGTM

**Pull request merged**